TCGA-17-Z058/DiagnosticReport/diagnostic_report.z058.ct0433.json

```diff
@@ -0,0 +1,71 @@
+// diagnostic_report.Z058.ct0433.json
+// id: Z058
```

Comment:
I agree regarding the file names, but the IDs cannot have punctuation inside them.  Check out the IDs for the BreastDx patients.